index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/bf723a76a7.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <div class="header">
+        <div class="header_elements name"><h3>ikpro</h3></div>
+        
+        <div class="header_elements icons"><i class="fa-solid fa-bars"></i></div>
+    </div>
+
+    <div class="intro">
+        <div><h1>Inayat-khan</h1></div>
+        <div><h2>Software Developer</h2></div>
+        <div>
+            <p>
+                I can help you build a product , feature or website 
+                Look through some of my work and experience! If 
+                you like what you see and have a project you need 
+                coded, don’t hestiate to contact me. 
+            </p>
+            
+        </div>
+    </div>
+
+    <div class="abilities">
+        <div class="languages">
+            <ul style="list-style-type: none">
+                <li class="title">Language</li>
+                <li>JavaScript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+                <li>C++</li>
+                <li>Python</li>
+                <li>flutter</li>
+                <li>java</li>
+                <li>C#</li>
+            </ul>
+        </div>
+        <div class="Framworks">
+            <ul style="list-style-type: none">
+                <li class="title">Frameworks</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+            </ul>
+            
+        </div>
+        <div class="Skills">
+            <ul style="list-style-type: none">
+                <li class="title">Skills</li>
+                <li>Codepen</li>
+                <li>Github</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+          
+        </div>
+
+    </div>
+
+    <hr>
+
+    <div class="container">
+        <i class="fa-brands fa-twitter"></i>
+        <i class="fa-brands fa-github-square"></i>
+        <i class="fa-brands fa-linkedin"></i>
+    </div>
```

Comment:
Kindly use a `ul` tag to group a list of icons and `li` per icon. Please make sure that each icon has a link (`a` tag) so that a user can be able to click it to be redirected to the particular social website.

-  Please remember to include an `href` on your `<a>` tags even if you don't have URLs yet. You can leave the `href` as `href="#"` if you do not have URLs.

Also:
Please try to stick to the Figma design as possible. Your social icons should be in the middle.
![image](https://user-images.githubusercontent.com/38088818/153521597-66945854-7633-4c3f-a735-26df18c6cc88.png)

Your icons:
![image](https://user-images.githubusercontent.com/38088818/153521537-678db40b-c69c-4f18-aefd-c9c2871e48a8.png)

Figma Icons:
![image](https://user-images.githubusercontent.com/38088818/153521698-b470caa7-3264-4a83-ab90-1c68e4298ba5.png)